packages/php-db-import-export/src/ImportOptionsInterface.php

```diff
@@ -41,4 +41,6 @@ public function importAsNull(): array;
      * @return string[]
      */
     public function features(): array;
+
+    public function nativeTypesFeatureAllowed(): bool;
```

Comment:
ty jo to je zavadejici, protoze pri povolenych nativnich typech to nemusi delat ten DISTINCT. je to na dve specificke ficury. Prejmenoval bych to na neco jako `compareAllColumnsInNativeTable` a prihodil asi maly komentar

Reply:
ok